6/debian9/6.3/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM gcr.io/google-appengine/debian9
+FROM marketplace.gcr.io/google/debian9
 ENV KIBANA_VERSION 6.3.2
 ENV KIBANA_SHA512 87236fa1a45a5c5e0ecdc033f864e9a1f74cb148e350b83c6c024b37eb71e3c16236fe52525898532f286f816cb6d03de23cba3d97e0cc0c4d622cdd18c7defb
 
-ENV C2D_RELEASE=${KIBANA_VERSION}
+ENV C2D_RELEASE 6.3.2
```

Comment:
Please use the env variable to avoid repetition of this version number. Alternatively, get rid of the KIBANA_VERSION env variable above

Reply:
This file is generated automatically from [Dockerfile.template](https://github.com/GoogleCloudPlatform/kibana-docker/pull/21/files#diff-dcc6557039ec3c55761d85c27b6ae525). I fixed this to explicitly expand this variable to avoid errors when solution uses non semantic version format. We use the same convention in other solutions.

Reply:
KIBANA_VERSION is a commonly used software version number, however C2D_RELEASE variable is project specific and used in our k8s applications, so I wouldn't get rid from the former